Backpack-SwiftUI/GraphicPromo/Classes/BPKGraphicPromo.swift

```diff
@@ -93,52 +116,83 @@ public struct BPKGraphicPromo: View {
     // swiftlint:disable closure_body_length
     @ViewBuilder
     private func contentView() -> some View {
-        VStack(alignment: .leading, spacing: .md) {
-            if verticalAlignment == .bottom {
-                if let sponsor {
-                    sponsorOverlayView(sponsor)
-                    Spacer()
-                } else {
-                    Spacer()
+        Group {
+            if let sponsor {
+                sponsorContentView(sponsor)
+            } else {
+                Spacer()
+                VStack(alignment: .leading, spacing: .md) {
+                    if verticalAlignment == .bottom {
+                        Spacer()
+                    }
+                    
+                    if let kicker {
+                        BPKText(kicker, style: .label1)
+                            .foregroundColor(variant.foregroundColor)
+                            .lineLimit(nil)
+                    }
+                    
+                    BPKText(headline, style: .heading2)
+                        .foregroundColor(variant.foregroundColor)
+                        .lineLimit(nil)
+
+                    if let subheadline {
+                        BPKText(subheadline, style: .heading5)
+                            .lineLimit(nil)
+                            .foregroundColor(variant.foregroundColor)
+                    }
+                    
+                    if verticalAlignment == .top {
+                        Spacer()
+                    }
                 }
             }
-            
-            if let kicker {
-                BPKText(kicker, style: .label1)
-                    .foregroundColor(variant.foregroundColor)
-                    .lineLimit(nil)
-            }
-            
-            BPKText(headline, style: .heading2)
+        }
+    }
+    
+    @ViewBuilder
+    private func sponsorContentView(_ sponsor: Sponsor) -> some View {
+        VStack(alignment: .leading, spacing: .md) {
+            Spacer()
+            BPKText(Text(headline).bold(), style: .heading2)
                 .foregroundColor(variant.foregroundColor)
                 .lineLimit(nil)
-
-            if let subheadline {
-                BPKText(subheadline, style: .heading5)
-                    .lineLimit(nil)
-                    .foregroundColor(variant.foregroundColor)
-            }
             
-            if verticalAlignment == .top {
-                if let sponsor {
-                    Spacer()
-                    sponsorOverlayView(sponsor)
-                } else {
-                    Spacer()
+            if #available(iOS 16.0, *) {
```

Comment:
I know our usage will be based off iOS 16, but what does this mean for iOS 15 usage within BPK context?
Should we make the sponsored init iOS 16 only as well? (Sponsored Graphic Promotion without sponsor section makes no sense)

Reply:
Let's add it for the init 👍